doc/usage.md

```diff
@@ -9,11 +9,20 @@ Use the `-t` option to run rules with the given metadata value (see the rule fie
 For example, `capa -t william.ballenthin@mandiant.com` runs rules that reference Willi's email address (probably as the author), or
 `capa -t communication` runs rules with the namespace `communication`.
 
+### only analyze selected functions
+Use the `--functions` option to extract capabilities from only a selected set of functions.
```

Comment:
```suggestion
Use the `--functions` option to extract capabilities from only a selected set of functions.
This is useful when XXX and we want YYY.
```

Reply:
help users quickly understand when they should use this option (such as data overload).